marathon-gradle-plugin/src/main/kotlin/com/malinskiy/marathon/ConfigurationFactory.kt

```diff
@@ -96,6 +97,8 @@ private fun createAndroidConfiguration(
             }
         }
         ?: SerialStrategy.AUTOMATIC
+    val cleanupDeviceScript = extension.cleanupDeviceScript
```

Comment:
Does this have to be customisable? or is nullability a way to disable the cleanup?

Reply:
I can put default variant here which will remove all test apks